code/unclassified/src/leap_year/leap_year.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import "fmt"
+
+func isLeapyear(year int) bool {
```

Comment:
Also, these functions should start with an uppercase letter, since I assume they are public functions.